README.md

```diff
@@ -177,239 +189,175 @@ sergio@sergio:~/plz/examples/pytorch$ plz history | \
-chmod u+x get-docker.sh
-./get-docker.sh
-sudo usermod -aG docker $USER
-
-# Install docker compose
-
-sudo curl -L https://github.com/docker/compose/releases/download/1.21.2/docker-compose-$(uname -s)-$(uname -m) -o /usr/local/bin/docker-compose
-sudo chmod +x /usr/local/bin/docker-compose
-
-# Clone plz
-
-git clone https://github.com/prodo-ai/plz.git
-
-cd plz
-
-# Install the cli
-
-./install_cli
-
-# Start a new terminal so that the current user is in the docker group,
-# and the plz executable is in the path (pip3 will put it in $HOME/.local/bin)
-
-sudo su - $USER
-
-# If you executed the previous command, you're not inside plz/ anymore
-
-cd plz
-
-# Check that docker works (you should see a possibly empty list of images,
-# and not a permission or connection error)
-
-docker ps
-
-# Start the local controller. Instructions for the controller that uses AWS are
-# detailed below. This runs detached and spits the output to the terminal.
-# You might want to run it in a different terminal (remember `cd plz/`).
-
 ./start_local_controller
-
 ```
 
 The controller can be stopped at any time with:
+
 ```
 ./stop_controller
 ```
 
-
 ### AWS configuration
 
-You need to do all the steps listed above for the local configuration, except
-for the last one starting the local controller. The additional steps for using
-AWS are below.
+To start a controller that talks to AWS, you'll need to first set up the AWS
+CLI:
 
-*Note: the command
-`./start_aws_controller` will take some time the first time it's run, as it
-downloads a whole pytorch environment to be used in docker (unless you've run
-the local configuration before) and also uploads that your AWS infrastructure
-so that it's ready for your instances to use*
+1. Install the AWS CLI: `pip install awscli`
+2. Configure it with your access key: `aws configure`
 
-*Note: if you usually use AWS in a particular region, please edit
-aws_config/config.json and set your region there. The default file sets the
-region to eu-west-1, Ireland.*
+If you usually use AWS in a particular region, please edit
+`aws_config/config.json` and set your region there. The default file sets the
+region to _eu-west-1_ (Ireland).
 
+Then run:
 
 ```
-# Configure access to AWS.
-# You'll need your access key. You can get it from the AWS console.
-# In the top bar, click on your name, then `My security credentials`, then
-# create one in `Access keys`
-# You only need to set the access key ID and the secret access key
-
-aws configure
-
-# Check that AWS works. You should see a possibly empty list of instances
-
-aws ec2 describe-instances --region eu-west-1
-
-# Start the AWS controller. This runs detached and spits the output to the
-# terminal. You might want to run it in a different terminal (remember
-# `cd plz/`). Remember to edit `aws_config/config.json` to set your region,
-# unless you want eu-west-1.
-
 ./start_aws_controller
 ```
 
-## Examples
+Unless you add `"instance_max_uptime_in_minutes": null,` to your
```

Comment:
I had put these notes at the beginning of the examples because a user that just wants to try the examples is more likely to see them. I still think it's a good place to put them. At the bottom of the installation instructions they can be easily forgotten / overlooked.

Reply:
Eh, it's not very far away, and totally irrelevant if you're not using AWS. And irrelevant for the example, because you don't need to train for more than 60 minutes. I agree that this is not the best place, but I don't see how the example is better.